copy_this/modules/fc/fcoxid2afterbuy/core/fco2aartimport.php

```diff
@@ -317,6 +320,55 @@ protected function _fcAddIdentificationData($oXmlProduct, &$oArticle, $sType)
         }
     }
 
+    /**
+     * declare articles as invalid if article number does not match the configurated conditions
+     *
+     * @param $oXmlProduct
+     * @return bool
+     */
+    protected function _fcCheckIfArticleNumberIsValid($oXmlProduct) {
+        $blDiscard = $this->getConfig()->getConfigParam('blFcAfterbuyIgnoreArticlesWithoutNr');
```

Comment:
Wouldn't this cause that EVERY article will be discarded if option is set to true?
I guess you wanted this combined with a check if there is an artnum...